.gitignore

```diff
@@ -0,0 +1,4 @@
+*./terraform*
+*.terraform.lock.hcl*
+*terraform.tfstate*
+*terraform*
```

Comment:
could you please, add newline at the end of file.

Reply:
Done